sec-policy/selinux-base-policy/selinux-base-policy-2.20210203-r1.ebuild

```diff
@@ -105,12 +107,26 @@ pkg_postinst() {
 		COMMAND="${COMMAND} -i ${i}.pp"
 	done
 
+	for i in ${OLD_MODS}; do
+		if [ -n "${MODS##*$i*}" ]; then
+			DEL_MODS="${DEL_MODS} ${i}"
```

Comment:
Once the line above is changed to just the list of modules to remove then this loop can be changed to die instead.

maybe something like:
```
for i in ${OLD_MODS}; do
    [[ ${MODS} != *${i}* ]] || die "Duplicate module in MODS and OLD_MODS: ${i}"
done
````